<?xml version="1.0"?>
<!-- ROS head file to simulate vizzy in gazebo -->
<launch>
	<!-- launch stereo_calibration -->
	<!--<node name="foveated_stereo_ros" pkg="foveated_stereo_ros" type="foveated_stereo_ros_node" respawn="false" output="screen" >
  	<rosparam file="$(find foveated_stereo_ros)/config/params.yaml" command="load" />
		<remap from="left_image" to="/vizzy/l_camera/image_raw"/>
		<remap from="right_image" to="/vizzy/r_camera/image_raw"/>
		<remap from="left_camera_info" to="/vizzy/l_camera/camera_info"/>
		<remap from="right_camera_info" to="/vizzy/r_camera/camera_info"/>
	</node>-->
	<!--<node name="ego_sphere_manager_ros" pkg="foveated_stereo_ros" type="ego_sphere_manager_ros_node" respawn="false" output="screen" >
  		<rosparam file="$(find foveated_stereo_ros)/config/egosphere.yaml" command="load" />
	</node>-->

	<!-- nodelet manager -->
  	<node pkg="nodelet" type="nodelet" name="nodelet_manager"
        	args="manager"
	    	respawn="true" output="screen"/>

  	<node pkg="nodelet" type="nodelet" name="foveated_stereo_ros"
        	args="load foveated_stereo_ros/FoveatedStereoRosNodelet nodelet_manager"
	    	respawn="true" output="screen">
  		<rosparam file="$(find foveated_stereo_ros)/config/params.yaml" command="load" />
		<remap from="left_image" to="/vizzy/l_camera/image_raw"/>
		<remap from="right_image" to="/vizzy/r_camera/image_raw"/>
		<remap from="left_camera_info" to="/vizzy/l_camera/camera_info"/>
		<remap from="right_camera_info" to="/vizzy/r_camera/camera_info"/>
	</node>

  	<node pkg="nodelet" type="nodelet" name="egosphere"
        	args="load foveated_stereo_ros/EgoSphereManagerRosNodelet nodelet_manager"
	    	respawn="true" output="screen">
  		<rosparam file="$(find foveated_stereo_ros)/config/egosphere.yaml" command="load" />
	</node>



</launch>
